### ISSUE TITLE
SCIFIO_Reader plugin or class not found

### ISSUE THREAD

**Issue by ngladitz** [fiji/fiji#305]
I noticed File -> Import -> SCIFIO... produces the error:
![grafik](https://user-images.githubusercontent.com/4358176/163157862-69186929-4653-4294-8b8f-3a495cc926f8.png)

I tried digging into where I might fix this.
Found out the class no longer exists and the menu entry has already been removed a while ago by IO_-4.2.0.
Fiji still only has IO_-4.1.0 available.

Is this expected due to some release process schedule or an oversight?

**Comment by ctrueden** [comment]
Thanks @ngladitz for the report. I have now uploaded IO_ version 4.2.0 to the Java-8 update site, to address this version skew.

The normal update process is that a new [pom-scijava bill of materials](https://github.com/scijava/pom-scijava) gets released after validating that all component versions work together successfully at runtime, and then these versions are uploaded to the core ImageJ update sites as appropriate. But we have been stalled on the next pom-scijava for a few months now due to a build- up of incompatible version updates. We are actively working on reconciling those so that we can get a new pom-scijava released and then a new Fiji with lots of component upgrades. As a stopgap until then, I uploaded the new `IO_`.
